Sources/Tropos/Views/DailyForecastView.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+import TroposCore
+
+@objc(TRDailyForecastView) public class DailyForecastView: UIView {
+    @IBOutlet weak var dayOfWeekLabel: UILabel!
+    @IBOutlet weak var highTemperatureLabel: UILabel!
+    @IBOutlet weak var lowTemperatureLabel: UILabel!
+    @IBOutlet weak var conditionsImageView: UIImageView!
+    @IBOutlet var contentView: DailyForecastView!
+    @objc public var viewModel: DailyForecastViewModel!
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    public override func awakeFromNib() {
+        super.awakeFromNib()
+        let mainBundle = Bundle.main
+        mainBundle.loadNibNamed(String(describing: "TRDailyForecastView"), owner: self, options: nil)
```

Comment:
I think this should either be `String(describing: type(of: self))`, `String(describing: DailyForecastView.self)`, or a literal string.

Reply:
I had some runtime errors when using the DailyForecastView in a description string. Wasn't sure what was happening at the time so put in a string in place of the class at the time to test but forgot to take it out. 

Just renamed the nib file to match the class name.

Reply:
> Just renamed the nib file to match the class name.

Excellent. `String(describing: DailyForecastView.self)` returns the name of the type as a string, i.e., `DailyForecastView`. `NSStringFromClass(DailyForecastView.self)` returns the Objective-C class name, which in this case is `TRDailyForecastView`, or if the name is not customised `<module>.<class>`, e.g. `Tropos.DailyForecastView`. It’s conventional to name the .xib after the class name, so `String(describing: DailyForecastView.self)` should work.